p2p/libp2p/connectionMonitor/nilConnectionMonitor.go

```diff
@@ -0,0 +1,41 @@
+package connectionMonitor
+
+import (
+	"fmt"
+
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/libp2p/go-libp2p-core/network"
+	"github.com/multiformats/go-multiaddr"
+)
+
+type NilConnectionMonitor struct {
```

Comment:
why is this mock needed ?

Reply:
To always have a non nil implementation. Nil connection monitor will have 0 performance impact when applied as its methods are empty.